tmail-backend/event-bus-redis/src/main/java/org/apache/james/events/CleanRedisEventBusService.java

```diff
@@ -0,0 +1,147 @@
+package org.apache.james.events;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.function.Function;
+
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+
+import org.apache.james.util.MDCStructuredLogger;
+import org.apache.james.util.ReactorUtils;
+import org.apache.james.util.StructuredLogger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import io.lettuce.core.KeyScanArgs;
+import io.lettuce.core.KeyScanCursor;
+import io.lettuce.core.ScanCursor;
+import io.lettuce.core.api.reactive.RedisKeyReactiveCommands;
+import io.lettuce.core.api.reactive.RedisSetReactiveCommands;
+import io.lettuce.core.pubsub.api.reactive.RedisPubSubReactiveCommands;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+public class CleanRedisEventBusService {
+    public static class Context {
```

Comment:
This "context" is a remain from the previous webadmin version of the code, and unused here.

Remove it?